db-async-common/src/main/java/com/github/jasync/sql/db/Configuration.kt

```diff
@@ -50,6 +51,7 @@ data class Configuration @JvmOverloads constructor(
     val maximumMessageSize: Int = 16777216,
     val allocator: ByteBufAllocator = PooledByteBufAllocator.DEFAULT,
     val connectionTimeout: Int = 5000,
+    val createTimeout: Duration = Duration.ofMillis(5000),
```

Comment:
I think this is exactly as connectionTimeout

Reply:
Indeed, will fix this